kotlin/android/app/src/main/java/dev/firezone/android/tunnel/TunnelService.kt

```diff
@@ -230,6 +256,7 @@ class TunnelService : VpnService() {
     private fun connect() {
         val token = appRestrictions.getString("token") ?: repo.getTokenSync()
         val config = repo.getConfigSync()
+        disabledResources = repo.getDisabledResourcesSync().toMutableSet()
```

Comment:
For favorites I had `get` return a `HashSet` directly here, although that means a pointless allocation if the set will never be mutated

Reply:
`getDisabledResourcesSync` returns a `Set` already but it's not mutable, I think the signature is clearer if the caller explicitly converts it to mutable.